### PR TITLE
Updates for 0.16

### LINKED ISSUE #95: Support for new LISTEN_ADDRESS Kolibri option
After https://github.com/learningequality/kolibri/pull/9291 is merged into develop, a new option `LISTEN_ADDRESS` can be added to the kolibri `options.ini` file.
Its default value keep things as they were until now. However, if the option is used, nginx conf should be modified changing
` listen 80; `and` listen 81; `lines to
`listen LISTEN_ADDRESS:80; ` and `LISTEN_ADDRESS:81;`

### LINKED ISSUE #96: Update kolibri-server build to use the error pages generated by kolibri
After https://github.com/learningequality/kolibri/pull/10162 is approved, the translation of the error pages has been centralized in kolibri.
kolibri-server must update its build process to use these pages instead of its own.
Also, kolibri-server project from Crowdin can be removed.

### PR DESCRIPTION
**Pull request by jredrejo**
This PR adapts kolibri-server to work with the new changes included in kolibri 0.16
Closes: #95, #96

File debian/changelog has the list of changes:
  * Bumped python minimum version to 3.6
  * Added kolibri build dependency to automate error pages creation (Closes: #96)
  * debian/rules:
    - modified to automate error pages creation
    - force gzip compression for old OS compatibility
  * debian/compat bumped to version 10
  * Improved README.rst file (thanks to Shrenik Bhura)
  * nginx.conf takes into account new kolibri LISTEN_ADDRESS option (Closes: #95)

### CODE REVIEW

File: debian/rules [hunk 1]
```diff
@@ -6,10 +6,11 @@
...
+	dh_builddeb -- -Zgzip
+
 override_dh_compress:
-	for file in $$(cd nginx_error_page/ && find -name "error.html" -printf '%P\n'); do \
-		mkdir -p $(CURDIR)/debian/kolibri-server/usr/share/kolibri/error_pages/$$(dirname $$file); \
-		sed 's/1.2.3/$(shell dpkg-parsechangelog -SVersion)/' \
-		nginx_error_page/$$file > $(CURDIR)/debian/kolibri-server/usr/share/kolibri/error_pages/$$file;\
-	done
+	# Create nginx error pages
+	kolibri manage loadingpage --reload --output-dir $(CURDIR)/debian/kolibri-server/usr/share/kolibri/error_pages --version-text $(shell dpkg-parsechangelog -SVersion)
```
Comment (rtibbles) [inline debian/rules:15]: Nice!

File: uwsgi.ini [hunk 2]
```diff
@@ -23,7 +23,7 @@ plugin = python3
 max-requests = 1000 # Reload workers after the specified amount of managed requests (avoid memory leaks).
 vacuum = True       # Try to remove all of the generated files/sockets (UNIX sockets and pidfiles) upon exit.
 module =kolibri.deployment.default.wsgi:application
-buffer-size = 8192  # needed to support long requests blocks (in lessons, for example)
+buffer-size = 32768  # needed to support long requests blocks (in lessons, for example)
```
Comment (rtibbles) [inline uwsgi.ini:26]: Presumably this is only an issue for large API requests, as file requests are streamed?

**Review by rtibbles — APPROVED** [review]
Makes sense to me.

I guess we can add the Github action for publishing new releases later :)

File: uwsgi.ini (hunk 2 above)
Comment (jredrejo) [reply uwsgi.ini:26]: it's thought in long urls (or posts with a lot of parameters, this kind of errors raised especially when accessing to lessons with many resource ids)

**Pull request merged**